### ISSUE TITLE
Argo CD unable to generate template with pulsar chart 3.3.0

### ISSUE THREAD

**Issue by yonatanamz** [apache/pulsar-helm-chart#471]
**Describe the bug**
When i am adding my pulsar chart to Argo CD as an Argo CD app its failing to render the template of the chart

**To Reproduce**
1. Clone the pulsar chart 
2. Create an Argo CD app with the pulsar chart

**Expected behavior**
Argo CD should be able to template the chart 

**Screenshots**
![Screenshot 2024-03-14 at 14 45 07](https://github.com/apache/pulsar-helm-chart/assets/30150811/764eb35b-a496-4928-a8e8-b71cc3abafaa)

**Comment by lhotari** [comment]
@yonatanamz do you have a chance to follow the instructions in the error message and check what the invalid YAML looks like? Please don't share screenshots of textual information since when it's an image, it's hard to copy-paste the error to do googling with the error message etc.

**Comment by yonatanamz** [comment]
Hi @lhotari I followed the instructions in the error message and ran the following command (I didn't get any errors)
`helm template --name-template pulsar --namespace pulsar-dev charts/pulsar -f charts/pulsar/values.yaml -f values/eu-dev-3/pulsar.yaml  --kube-version 1.24 --include-crds --debug > pulsar-dev-template-debug.yaml`

This is the rendered autorecovery-configmap that i got:
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: "pulsar-recovery"
  namespace: pulsar-dev
  labels:
    app: pulsar
    chart: pulsar-3.3.0
    release: pulsar
    heritage: Helm
    cluster: pulsar
    component: recovery
data:
  # common config
  zkServers: "pulsar-zookeeper:2181"
  zkLedgersRootPath: "/ledgers"
  # enable bookkeeper http server
  httpServerEnabled: "true"
  httpServerPort: "8000"
  # config the stats provider
  statsProviderClass: org.apache.bookkeeper.stats.prometheus.PrometheusMetricsProvider
  # use hostname as the bookie id
  useHostNameAsBookieID: "true"
  BOOKIE_MEM: |
    -Xms64m -Xmx64m
  PULSAR_PREFIX_useV2WireProtocol: "true"
---
```

**Comment by lhotari** [comment]
@yonatanamz there are fixes in the master branch to support Argo CD. Please check PRs #480 and #478 for details.

**Comment by lhotari** [comment]
@yonatanamz the fixes will be included in the upcoming 3.4.0 version of the Apache Pulsar Helm chart. This is currently in release candidate voting stage. Please help validate the Apache Pulsar Helm chart release 3.4.0: https://lists.apache.org/thread/t4vlvk0nwd3gzo50t466cqy5h95l81gb . The release vote email contains instructions how to use the release candidate version.

**Comment by hpvd** [comment]
3.4.0 version of the Apache Pulsar Helm chart is now available:
https://github.com/apache/pulsar-helm-chart/releases/tag/pulsar-3.4.0
